tests/search.py

```diff
@@ -95,3 +104,61 @@ def get_model_full_text(cls, model):
     @classmethod
     def get_model_full_text_boosted(cls, model):
         return model.title
+
+
+class DefaultNewSearchDocTypeMixin(GenericDocType):
+    """
+    Used by get_new_search_index() in the case that doc_type_mixin is not supplied
+    """
+
+    @classmethod
+    def get_queryset(cls, last_updated_datetime=None):
+        qs = Movie.objects.all()
+        if last_updated_datetime:
+            qs = qs.filter(last_modified__gte=last_updated_datetime)
+        return qs
+
+
+@contextmanager
+def get_new_search_index(name, doc_type_mixin=None, create_and_activate=True, dem_index=None):
```

Comment:
This test helper context manager creates a new transient index that can be iteratively mutated so as to test how the library handles changes in a codebase